documentation/setup/download.mdx

```diff
@@ -25,6 +25,7 @@ If you are not sure what type of Home Assistant you are using or how to find tha
 ```bash
 wget -O - https://get.hacs.xyz | bash -
 ```
+After the script is completed, remember to restart Home Assistant.
```

Comment:
That's fair :) 

Just thought it would make sense to include it in the instructions followed when installing, so it wouldn't be missed if not seeing it in the console.